dla_cnn/desi/DesiMock.py

```diff
@@ -88,6 +95,18 @@ def get_data(start_point = 0, end_point = self.data_size):
             sightline.dec = self.data[id]['DEC']
             sightline.dlas = self.data[id]['DLAS']
             sightline.loglam = np.log10(self.wavelength[start_point:end_point])
+
+            wavelength = self.wavelength[start_point:end_point]
+            dlambda = wavelength-np.roll(wavelength,1)
```

Comment:
Please explain what these lines of code are for.
I'll note there will always be at least 1 negative value because `np.roll` wraps the array around.

Reply:
Emmm, I just chose the whole array at this place. And I did the slicing when I really needed it like this ` test = (dlambda>0)[1:]` . Sorry for confusing you.